pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentSubscription.java

```diff
@@ -131,6 +133,9 @@ public class PersistentSubscription extends AbstractSubscription implements Subs
     private volatile Map<String, String> subscriptionProperties;
     private volatile CompletableFuture<Void> fenceFuture;
 
+    @Getter
+    private volatile double filterAcceptedRateEstimation;
```

Comment:
What do you think about defaulting this to 1? I think it would be preferred to overestimate the backlog.

Reply:
1 means that  every message is accepted, and that's the case in which you have no filter

the default is set when initializing the object

Reply:
Is it initialized on line 1373? I don't see it set when the object is initialized.